s3/s3.go

```diff
@@ -0,0 +1,172 @@
+package s3
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	bindings "github.com/andream16/aws-sdk-go-bindings"
+	"github.com/andream16/aws-sdk-go-bindings/internal/format"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"github.com/pkg/errors"
+)
+
+// S3er describes s3 API.
+type S3er interface {
+	CreateBucket(string) error
+	GetObject(bucket, path string) ([]byte, error)
+	PutObject(bucket, objectName, objectPath string) error
```

Comment:
Minor: given the method includes "object" I'd drop it from the `PutObject` params; it becomes slightly more concise - plus you already name it that way for `GetObject`. Also, I'd put `path` before `name` for two reasons: it should go from lower resolution to more specific (on a FS: bucket (drive) -> path (directory) > name (filename)) - it would also mirror `GetObject`'s param order, which IMO is better ergonomically.

```go
type S3er interface {
	CreateBucket(string) error
	GetObject(bucket, path string) ([]byte, error)
	PutObject(bucket, path, name string) error
}
```